src/vnsw/agent/contrail/SConscript

```diff
@@ -104,8 +104,7 @@ if sys.platform.startswith('freebsd'):
 elif sys.platform.startswith('linux'):
     platform_dependent = 'linux/pkt0_interface.cc'
 elif sys.platform.startswith('win'):
-    platform_dependent = ['windows/pkt0_interface.cc','#build/lib/windowsstubs.lib']
-
+    platform_dependent = ['windows/pkt0_interface.cc','#build/lib/windowsstubs.lib', env.RES('windows/resource.rc')]
```

Comment:
It is in windows directory. Specific to windows only. Only if a new resource is added,someone will need to modify it.